docs/switch.md

```diff
@@ -17,6 +17,9 @@ Switches are simple representations of binary actors. They mainly support switch
 - `name` is the name of the object.
 - `group_address` is the KNX group address of the switch device. Used for sending.
 - `group_address_state` is the KNX group address of the switch state. Used for updating and reading state.
+- `group_address_current_power` is the KNX group address of the current power usage.
+- `group_address_total_energy` is the KNX group address of the total energy consumed.
+- `group_address_standby` is the KNX group address to indicate if the device connected to the switch is currently in standby.
```

Comment:
- You added `sync_state` attribute. Please mention it here too.
-  Please add DPT for each group_address - see eg. fan.md

Reply:
And perhaps also mention the unit. I see W and kWh in the code below.

Reply:
Added in latest commit.

Reply:
If we leave `group_address_standby` in I'd suggest to rename it `group_address_standby_state` so it is clear that is is a read-only value and will be read by StateUpdater. 
Maybe same for the other 2 but energy metering kind of implies this anyway.